scripts/eventbrite-transformations.py

```diff
@@ -0,0 +1,78 @@
+import requests
+import mysql.connector as mysql
+import configparser
+import json
+
+### read-in data need for sql connection
+config = configparser.ConfigParser()
+config.read('../config.ini')
+
+db = mysql.connect(
+    host=config['mysql']['host'],
+    user=config['mysql']['user'],
+    passwd=config['mysql']['pass'],
+    database="state_of_music"
+)
+cursor = db.cursor(buffered=True)
+cursor2 = db.cursor(buffered=True)
+
+### stage genreDict.json for later user
+with open('genreDict.json','r',encoding="utf-8") as file:
+    genre_dict = json.loads(file.read())
+
+### run transformations
+query = """SELECT genre, eventbrite_id
+        FROM eventbrite_events"""
+
+cursor.execute(query)
+
+for row in cursor:
+    eb_genre = row[0].lower()
+    eb_id = row[1]
+
+    if eb_genre != '':
+        event_genre_dict = {'country':0,
```

Comment:
[SC]
You can initialize this dictionary in a more flexible way by first creating a list with all the genres and then iterating through it:

```python
genres = ['country', 'electronic music', 'folk', 'hip hop', 'jazz', 'pop', 'r&b and soul', 'rock', 'classical music']
event_genre_dict = {}
for genre in genres:
  event_genre_dict[genre] = 0
```